nemo_text_processing/text_normalization/en/taggers/electronic.py

```diff
@@ -76,60 +85,55 @@ def __init__(self, cardinal: GraphFst, deterministic: bool = True):
             NEMO_ALPHA | numbers | accepted_symbols | dict_words_graph
         )
 
-        username = pynutil.insert("username: \"") + username + pynutil.insert("\"") + pynini.cross('@', ' ')
+        username = insert_username + username + insert_double_quotes + pynini.cross("@", " ")
 
         domain_graph = all_accepted_symbols_start + pynini.closure(
             all_accepted_symbols_end | pynutil.add_weight(accepted_common_domains, MIN_NEG_WEIGHT)
         )
 
-        protocol_symbols = pynini.closure((graph_symbols | pynini.cross(":", "colon")) + pynutil.insert(" "))
+        protocol_symbols = pynini.closure((graph_symbols | pynini.cross(":", "colon")) + insert_space)
         protocol_start = (pynini.cross("https", "HTTPS ") | pynini.cross("http", "HTTP ")) + (
-            pynini.accep("://") @ protocol_symbols
+            accept_colon_double_slash @ protocol_symbols
         )
-        protocol_file_start = pynini.accep("file") + insert_space + (pynini.accep(":///") @ protocol_symbols)
+        protocol_file_start = accept_file + insert_space + (accept_colon_triple_slash @ protocol_symbols)
 
-        protocol_end = pynutil.add_weight(pynini.cross("www", "WWW ") + pynini.accep(".") @ protocol_symbols, -1000)
+        protocol_end = pynutil.add_weight(pynini.cross("www", "WWW ") + accept_period @ protocol_symbols, -1000)
```

Comment:
Why does the capitalized form have a space?

Reply:
so that you dont get things like `WWWdot nvidia dot com` ?